shared/rakefiles/vars.rb

```diff
@@ -51,6 +51,13 @@ def self.set_vars(env, project_type)
     # Hack to force Terraform to reapply some resources on every run
     ENV["TF_VAR_nonce"] = SecureRandom.hex
 
+    # Set HELM TLS variables
+    ENV["HELM_TLS_CERT"]    = "/project/live/#{env}/secrets/kube-system/helm-tls/helm.cert.pem"
```

Comment:
Should these vars be also covered by vars specs?

Reply:
I have added tests to cover `HELM_TLS_*` vars to `vars_spec.rb`